api/src/wfl/environment.clj

```diff
@@ -69,5 +69,4 @@
 (defn getenv
   "Lookup the value of the environment variable specified by `name`."
   [name]
-  (log/debugf "Reading environment variable %s" name)
```

Comment:
This is useful for detecting when we were doing IO when the namespace was being loaded.

Reply:
The log block starting with `01:19:15` in the PR description gives a sense of this log's volume and repetition.  Any time we spin up a transaction we get this:

```
01:19:15 DEBUG wfl.environment - Reading environment variable WFL_POSTGRES_URL
01:19:15 DEBUG wfl.environment - Reading environment variable WFL_POSTGRES_PASSWORD
01:19:15 DEBUG wfl.environment - Reading environment variable WFL_POSTGRES_USERNAME
01:19:15 DEBUG wfl.environment - Reading environment variable USER
```

So in the course of that run of the update loop, we saw this block 9 times.

If you think it has value, I can downgrade the level rather than remove completely.

Reply:
Yes please - make it trace.